src/config.ts

```diff
@@ -200,12 +234,10 @@ export interface Config {
   onUncaughtException?: string;
 
   /**
-   * Setting this to true or false is the same as setting contextHeaderBehavior
-   * to 'ignore' or 'default' respectively. If both are explicitly set,
-   * contextHeaderBehavior will be prioritized over this value.
-   * Deprecated: This option will be removed in a future release.
+   * Whether or not to use incoming trace context headers for distributed
+   * trace context propagation.
```

Comment:
Can you illustrate what happens in either case? How is `contextHeaderBehavior` used when this is `false`?

Reply:
I've removed this field.